tests/helpers/setup-no-deprecations.js

```diff
@@ -36,14 +36,16 @@ export default function setupNoDeprecations({ beforeEach, afterEach }) {
   };
 
   QUnit.assert.deprecationsInclude = function(expected) {
-    let foundIndex = deprecations.indexOf(expected);
+    let found = deprecations.find((deprecation) => deprecation.includes(expected));
```

Comment:
Matching by a substring wasn't possible before but needed due to the long deprecation messages introduced.